app-laptop/tpacpi-bat/files/tpacpi-bat.confd.0

```diff
@@ -3,6 +3,9 @@
 # use '/etc/init.d/tpacpi-bat info' at any time to check the thresholds
 # currently used by the driver
 
+# number of batteries you have
```

Comment:
I've understood the current purpose of `BATS` variable you define in the init, but I don't find any logic/simple use case:
For me, if someone have several batteries, I think he would set thresholds for all of them; perhaps with different thresholds, but for that, he has to modify the config (`BATS` and thresholds variables) or run the underlying command manually for batteries that don't respect global settings.

If defining thresholds per battery is common, shouldn't we find a proper way to do that?

```
BATS="1 3 4"

# Global thresholds
TPACPI_BAT_THRESH_START="30"
TPACPI_BAT_THRESH_STOP="85"

# Per battery thresholds
TPACPI_BAT_THRESH_START_BAT1="90"
TPACPI_BAT_THRESH_STOP_BAT1="99"

...
```